spec/controllers/hub/sites_controller_spec.rb

```diff
@@ -101,7 +128,8 @@
         site.reload
         expect(site.name).to eq "Silly Site"
         expect(site.parent_organization).to eq other_organization
-        expect(response).to redirect_to edit_hub_organization_path(id: other_organization)
+        expect(site.source_parameters.pluck(:code)).to eq(["newshortlink"])
```

Comment:
Consider adding an assertion that the old record was deleted (for example) to gain code coverage on the `:id` part of the change to the controller.

Reply:
Done; good call